tests/integration/test_code_blocks.py

```diff
@@ -20,6 +20,8 @@
     "docs/griptape-framework/structures/src/observability_1.py",
     "docs/griptape-framework/structures/src/observability_2.py",
     "docs/griptape-framework/data/src/loaders_9.py",
+    "docs/griptape-framework/data/src/loaders_10.py",
+    "docs/griptape-framework/drivers/src/web_search_drivers_3.py",
```

Comment:
Were these missing from the integration tests? Do we not have an automatic way of detecting them? (Like `docs/**/*.py` or something?)

Reply:
These particular tests are skipped because they rely on resources that are not trivial to set up (or are non-functional in the case of DDG)